feature-based-molecular-networking/Makefile

```diff
@@ -4,4 +4,4 @@ include ../Makefile.deploytemplate
 WORKFLOW_NAME=feature-based-molecular-networking
 TOOL_FOLDER_NAME=feature-based-molecular-networking
 WORKFLOW_VERSION=release_21
-WORKFLOW_DESCRIPTION="Feature-Based Molecular Networking (FBMN) is a computational method that bridges popular mass spectrometry data processing tools for LC-MS/MS and molecular networking analysis on GNPS. The supported tools are: MZmine, OpenMS, MS-DIAL, MetaboScape, XCMS, Progenesis QI, and the mzTab-M format. FBMN is better when quantification analysis is necessary, but requires preprocessing with feature finding tools. Classical molecular networking is better for discovery and can be analyzed directly from raw mass spectrometry files."
+WORKFLOW_DESCRIPTION='Feature-Based Molecular Networking (FBMN) is a computational method that bridges popular mass spectrometry data processing tools for LC-MS/MS and molecular networking analysis on GNPS. Please cite <a href="https://doi.org/10.1101/812404">Nothias LF et al. bioRxiv 2019</a> and <a href="https://doi.org/10.1038/nbt.3597"> Wang M et al. Nat. Biotech. 2016</a>. The supported tools are: MZmine, OpenMS, MS-DIAL, MetaboScape, XCMS, Progenesis QI, and the mzTab-M format. See the documentation at <a href="https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/">https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/</a>. FBMN facilitates the detection of isomers that are separated by chromatographic or ion mobility separation, and provides accurate ion abundances for statistical analysis. Note that FBMN requires processing the mass spectrometry data with a feature detection and alignment tool. For rapid/qualitative analysis, we recommend using classical molecular networking that accepts unprocessed mass spectrometry files.'
```

Comment:
I dont think we should but the citation in the description, in our deployment, we should specify a citation section

Reply:
Also, I don't know if single quotes is acceptable in Make sytax

Reply:
Yeah, I was wondering about single/double quotes.
The limitation is that people don't look at the citations section ... and the papers help to understand the tools.
At least, it would make sense to leave a link to the documentation for the workflow, if possible. Just do as you want it is fine.

Reply:
Am checking with Ben on how we want to handle this. I like having a uniform citation section, but we'll see where he stands on it as well.

Reply:
I did the change